microsoft/vscode
# Save file dialog: sort file types alphabetically

This PR fixes #114264

## How to Test

Create a new file and click File > Save or Save As..., then check order of file types.

## Result

### Before
![before](https://user-images.githubusercontent.com/11912225/104843020-abba4600-591c-11eb-8472-e42b52a94fe2.jpg)

### After
![after](https://user-images.githubusercontent.com/11912225/104843025-af4dcd00-591c-11eb-8191-c2f33bd7a9d2.jpg)
